.github/workflows/ci.yml

```diff
@@ -180,30 +191,55 @@ jobs:
+          git tag ${{ needs.get-versions.outputs.version }} --force
           git push --tags --force
 
-
+  # -----------------------------
+  # | PRERELEASE DEPLOYMENT JOB |
+  # -----------------------------
   prerelease-deploy:
     name: Deploy to Prerelease
-    # This will create a `spack` environment with the name `access-om2-<version>-<commit number>`
-    # For example, `access-om2-2024.01.1-3` for the deployment based on the third commit on this
-    # `pre-2024.01.1` PR branch.
+    # This will create a `spack` environment with the name `access-om2-<version>-<commit number>`.
+    # For example, `access-om2-2024_01_1-3` for the deployment based on the third commit on the PR branch.
     needs:
-      - prerelease-deploy-version
-      - spack-yaml-checks
-      - validate
-      - check-versions-exist
-    # The conditional below asserts that we should run this job as long as the result of the dependent jobs
-    # are not 'failures' or 'cancelled' (aka, either 'success' or 'skipped'). Skipped jobs may be created
-    # if certain files are not updated (for example, we don't need to check the validity of json if none
-    # has been updated in the PR), so we should treat 'skipped' jobs as a vacuous 'success'.
-    # The `always()` status function is there, because (if we do not give a different status function)
-    # by default there is an implicit 'if: success()' which would short-curcuit and skip the job.
-    if: always() && !contains(needs.*.result, 'failure') && !contains(needs.*.result, 'cancelled')
+      - get-versions  # so we can give an appropriate version to the prerelease build
+      - update-prerelease-tag  # implies all the spack.yaml-related checks have passed
+      - check-json  # implies all the json-related checks have passed
     uses: access-nri/build-cd/.github/workflows/deploy-1-setup.yml@main
     with:
       type: prerelease
       ref: ${{ github.head_ref }}
-      version: ${{ needs.prerelease-deploy-version.outputs.version-build }}
+      version: ${{ needs.get-versions.outputs.version-build }}
     secrets: inherit
+
+  notifier:
+    name: Notifier
+    runs-on: ubuntu-latest
+    needs:
+      - check-json
+      - get-versions
+    permissions:
+      pull-requests: write
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.head_ref }}
+          fetch-depth: 0
+
+      - name: Comment
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GH_REPO: ${{ github.repository }}
+          BODY: |
+            This `${{ github.repository }}` model will be deployed with the following versions:
+            * `${{ needs.get-versions.outputs.version }}` as a Release (when merged).
+            * `${{ needs.get-versions.outputs.version-build }}` as a Prerelease (during this PR). This can be accessed on `Gadi` via `spack` at `/g/data/vk83/prerelease/apps/spack/0.20/spack` once deployed.
```

Comment:
If you wanted to be really fancy could we use `${{ needs.check-json.outputs.spack-config-version }}` in the path instead of hard-coding `0.20`?

Reply:
Unfortunately not - that is the `spack-config` repository version, which would be something like `2023.11.24`. But, we when we start supporting deployment to multiple versions of spack, we will update this to be generic